src/components/BrewerySearch.js

```diff
@@ -1,12 +1,14 @@
 import React, { Component } from 'react'
-import PropTypes from 'prop-types'
```

Comment:
1) Why are you removing the PropTypes? This is needed to verify that the BrewerySearch Component receives the props it requires.